postgres-connector/src/main/java/org/odpi/openmetadata/adapters/connectors/integration/postgres/ffdc/PostgresConnectorErrorCode.java

```diff
@@ -80,8 +80,31 @@
             "The Postgres Connector experienced an exception accessing the Postgres columns.",
             "Ensure that the Postgres database server is available.",
             "Verify that the Postgres database server is available."),
+
+    ERROR_REMOVING_COLUMNS( 400, "POSTGRES-CONNECTOR-400-011",
+            "The Postgres Connector experienced an exception while trying to remove a database entity from Egeria.",
+            "Ensure that the Postgres database server is available.",
+            "Verify that the Postgres database server is available."),
+
+
+    ERROR_REMOVING_DATABASES( 400, "POSTGRES-CONNECTOR-400-011",
+            "The Postgres Connector experienced an exception while trying to remove a database entity from Egeria.",
+            "Ensure that the Postgres database server is available.",
+            "Verify that the Postgres database server is available."),
+
+    PROPERTY_SERVER_EXCEPTION( 400, "POSTGRES-CONNECTOR-400-012",
+            "The Egeria OMAS Server returned a Property Server Exception",
+            "Verify that the Egeria OMAS server is available",
+            "Verify that the Postgres database server is available."),
+
+    CONNECTOR_CHECKED( 400, "POSTGRES-CONNECTOR-400-013",
+            "The Egeria OMAS Server returned a Connector Exception",
+            "Verify that the Egeria OMAS server is available",
+            "Check the Exception details to identify the issue"),
```

Comment:
consider activating the unique message checking that is in the core egeria build these messages would fail due to the duplicated user actions and the like